### PR TITLE
Shim for JQuery Plugin tag-it

### PR DESCRIPTION
**Pull request by rajvirtual**
shim for https://github.com/aehlke/tag-it

### CODE REVIEW

File: package-overrides/github/tagit/tagit@1.0.0.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "registry": "jspm", 
+  "main": "js/tag-it", 
+  "shim": { "js/tag-it": 
+  { "deps": ["jquery","jquery-ui","../css/jquery.tagit.css!"],
+     "exports":"$" } }, 
+  "dependencies": { "jquery": "github:components/jquery",
+                 "jquery-ui":"github:components/jqueryui@1.11.4",
```
Comment (guybedford) [inline package-overrides/github/tagit/tagit@1.0.0.json]: Better to use a semver range here - `@^1.11.4`

File: tag-it [hunk 2]
```diff
@@ -0,0 +1,11 @@
+"{
```
Comment (guybedford) [inline tag-it]: This file isn't necessary

**Comment by rajvirtual** [comment]
Renamed file for semvar

**Comment by rajvirtual** [comment]
deleted old file.

File: package-overrides/github/tagit/tagit@^1.11.4.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "registry": "jspm", 
+  "main": "js/tag-it", 
+  "shim": { "js/tag-it": 
+  { "deps": ["jquery","jquery-ui","../css/jquery.tagit.css!"],
+     "exports":"$" } }, 
+  "dependencies": { "jquery": "github:components/jquery",
+                 "jquery-ui":"github:components/jqueryui@1.11.4",
```
Comment (guybedford) [inline package-overrides/github/tagit/tagit@^1.11.4.json]: This still needs to be a semver range - `@^1.11.4`.

Comment (rajvirtual) [inline package-overrides/github/tagit/tagit@^1.11.4.json]: would it be okay if i replace those dependencies lines with  
"dependencies": { "jquery": "github:components/jquery",
                              "jquery-ui":"jspm:jqueryui@^1.8"

Comment (guybedford) [inline package-overrides/github/tagit/tagit@^1.11.4.json]: Sure!

Comment (guybedford) [inline package-overrides/github/tagit/tagit@^1.11.4.json]: Even better would be `"jquery": "*"` actually.

**Comment by rajvirtual** [comment]
Done !

**Pull request merged**